mos_tests/rabbitmq_oslo/test_restarts.py

```diff
@@ -236,6 +236,33 @@ def consume_msg(remote, cfg_file_path):
     num_of_msg_consumed = int(re.findall('\d+', out_consume)[0])
     return num_of_msg_consumed
 
+
+def rabbit_rpc_server_start(remote, cfg_file_path):
+    logger.debug('Start [oslo_msg_check_server] on %s.' % remote.host)
+    background = '<&- >/dev/null 2>&1 &'
+    cmd = 'oslo_msg_check_server --nodebug --config-file {0} {1}'.format(
+        cfg_file_path, background)
+    remote.execute(cmd)
```

Comment:
AFAIK, this tests marked as undestructive, so you need to stop this processes after test.